app/models/patient.rb

```diff
@@ -73,4 +74,9 @@ def computed_age
     age -= 1 if Date.today < dob + age.years
     return age
   end
+
+  def dob_is_valid_date
+    errors.add(:dob, 'must be a valid date') if (!dob.is_a?(Date))
```

Comment:
Can we add a model test for this into 

  https://github.com/department-of-veterans-affairs/SCIInterim/blob/master/spec/models/patient_spec.rb

?